docs/backends/spdk/index.rst

```diff
@@ -0,0 +1,365 @@
+.. _sec-backends-spdk:
+
+SPDK
+====
+
+**xNVMe** provides a kernel-bypassing backend implemented
+using :xref-spdk:`SPDK<>`. **SPDK** has a lot to offer, however, **xNVMe**
+only makes use of the :xref-spdk:`SPDK<>` user-space NVMe driver. That is, the
+reactor, threading-model, and application-framework of **SPDK** is not used
+by **xNVMe**.
+
+Note, that **SPDK** in turn makes use of **xNVMe**, via the **bdev_xnvme**.
+
+.. _sec-backends-spdk-identifiers:
+
+Device Identifiers
+------------------
+
+Since devices are no longer available in ``/dev``, then the PCI ids are used,
+such as ``0000:03:00.0``, e.g. using the CLI:
+
+.. literalinclude:: 115_xnvme_info.cmd
+   :language: bash
+
+And using the API it would be similar to::
+
+  ...
+  struct xnvme_dev *dev = xnvme_dev_open("0000:01:00.0", opts);
+  ...
+
+Since the Kernel NVMe driver is unbound from the device, then the kernel no
+longer know that the PCIe device is an NVMe device, thus, it no longer lives in
+Linux devfs, that is, no longer available in ``/dev`` as e.g. ``/dev/nvme0n1``.
+
+Instead of the filepath in devfs, then you use PCI ids and xNVMe options.
```

Comment:
This is the same text twice

Reply:
Please skip the backends and tools sections, as they are incomplete and may not make sense. These sections were not addressed due to the substantial size of this PR.